components/account/WalletSelect.tsx

```diff
@@ -63,9 +63,10 @@ const WalletSelect = () => {
               )}
               {hasError && (
                 <div className="text-vermilion ml-auto  text-ztg-12-120 w-ztg-275">
-                  {error.type === "NoAccounts"
```

Comment:
What does this actually change? Since error type should only be one of two types it seems like the old code should cover all cases?

Reply:
I suspect the types are probably wrong and not all errors are accounted for. Even so, I don't think the previous code is good practice as it assumes that there will only ever be 2 error messages